package.json

```diff
@@ -78,13 +78,30 @@
                 },
                 "auto-close-tag.excludedTags": {
                     "type": "array",
-                    "default": ["area", "base", "br", "col", "command", "embed", "hr", "img", "input", "keygen", "link", "meta", "param", "source", "track", "wbr"],
+                    "default": [
+                        "area",
+                        "base",
+                        "br",
+                        "col",
+                        "command",
+                        "embed",
+                        "hr",
+                        "img",
+                        "input",
+                        "keygen",
+                        "link",
+                        "meta",
+                        "param",
+                        "source",
+                        "track",
+                        "wbr"
+                    ],
                     "description": "Set the tag list that would not be auto closed.",
                     "scope": "resource"
                 },
                 "auto-close-tag.SublimeText3Mode": {
                     "type": "boolean",
-                    "default": false,
+                    "default": true,
```

Comment:
We'd not change the default setting if we could not make sure most of users want this.

Reply:
I want this!